.github/workflows/build.yml

```diff
@@ -165,7 +190,7 @@ jobs:
 
           extract_file ./artifacts/libddwaf-linux-x86_64/libddwaf-*-linux-x86_64*.tar.gz \
             '/libddwaf\.so$' ./packages/nuget/runtimes/linux-x64/native
-          extract_file ./artifacts/libddwaf-macos/libddwaf-*-darwin-x86_64*.zip \
+          extract_file ./artifacts/libddwaf-macos-x86_64/libddwaf-*-darwin-x86_64*.zip \
```

Comment:
Maybe one should also add `libddwaf-macos-arm64` at some point, but I'm not sure if .Net supports it yet so I left it at `x86_64`.